static/js/browser2/observation_chart.tsx

```diff
@@ -94,28 +99,21 @@ export class ObservationChart extends React.Component<
         value: Number(values[key]),
       });
     });
-    if (data.length > MAX_DOTS) {
-      document
-        .getElementById("svg-container" + this.props.idx)
-        .classList.add("hide-dots");
-    }
-    const dataGroups = [new DataGroup("", data)];
     const svgContainerId: string = "svg-container" + this.props.idx;
```

Comment:
can make "svgContainerId" a private variable, then can remove it from the argument of addTooltip() and handleDotHighlight()

Reply:
done